code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -152,6 +152,8 @@
 	max_n2 = 0
 	minbodytemp = 0
 
+/mob/living/simple_animal/hostile/viscerator/emp_act(severity)
+	health -= 60*severity
```

Comment:
Wouldn't that kill it outright in any case? You can just make it destroy itself rather than damage

Reply:
the idea was making EMP instantly destroy the viscerators tbh, talked it with fortune